apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/SerializationTestData.cs

```diff
@@ -460,5 +469,20 @@ public CustomValueType FromFirestore(object value)
                     ["Value"] = value.Value
                 };
         }
+
+        [FirestoreData]
+        internal struct StructModel : IEquatable<StructModel>
+        {
+            [FirestoreProperty]
+            public string Name { get; set; }
+            [FirestoreProperty]
+            public int Value { get; set; }
+
+            public override int GetHashCode() => Name.GetHashCode() + Value;
+            public override bool Equals(object obj) => obj is StructModel other && Equals(other);
+            public bool Equals(StructModel other) => Name == other.Name && Value == other.Value;
+
```

Comment:
nit: Inconsistent blank line compared with `CustomValueType` above.

Reply:
Done - and added ToString to CustomPlayer too.